internal/commands/build/build.go

```diff
@@ -45,20 +45,21 @@ func Cmd(dockerCli command.Cli) *cobra.Command {
 	var opts buildOptions
 	cmd := &cobra.Command{
 		Use:     "build [OPTIONS] [CONTEXT_PATH]",
```

Comment:
```suggestion
		Use:     "build [OPTIONS] CONTEXT_PATH",
```

Because `CONTEXT_PATH` is mandatory

Reply:
APP-312 created to check all the commands and follow the pattern you talked about